sys/Makefile.dep

```diff
@@ -42,25 +42,37 @@ ifneq (,$(filter i2c_scan,$(USEMODULE)))
 endif
 
 ifneq (,$(filter prng_fortuna,$(USEMODULE)))
-  USEMODULE += crypto_aes
+  USEMODULE += crypto_aes_128
 endif
 
-ifneq (,$(filter crypto_aes_%,$(USEMODULE)))
-  USEMODULE += crypto_aes
+ifneq (,$(filter crypto_aes_unroll,$(USEMODULE)))
+  USEMODULE += crypto_aes_128
+endif
+
+ifneq (,$(filter crypto_aes_precalculated,$(USEMODULE)))
+  USEMODULE += crypto_aes_128
```

Comment:
These lines actually don't matter at all I think since `crypto_aes_128` is a default module if `crypto` is used and crypto is included based on `crypto_%`.